torch/nn/modules/upsampling.py

```diff
@@ -7,17 +7,13 @@
 
 class Upsample(Module):
     r"""Upsamples a given multi-channel 1D (temporal), 2D (spatial) or 3D (volumetric) data.
-
```

Comment:
Are these newline changes necessary?

Reply:
Sorry, they are not necessary. I must have used a different version of upsampling.py. The main change I wanted to make, is adding align_corners parameter for UpsamplingBilinear2d.